deploy_stack.ps1

```diff
@@ -0,0 +1,11 @@
+#!ps1
+
+if (Get-Command docker -errorAction SilentlyContinue)
+{
+    Write-Host "Deploying stack"
+    docker stack deploy func --compose-file ./docker-compose.yml
+}
+else

```

Comment:
Yeah good point, I can add update the bash scripts as well, would be tonight tho.